components/resizable-iframe/index.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * Imported from Calypso, with some lint fixes and gutenburg specific changes.
+ */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import debugFactory from 'debug';
```

Comment:
`debug` is not declared as a dependency. In fact, I'd suggest removing all this debug logic altogether.